tests/components/deconz/test_cover.py

```diff
@@ -81,6 +82,7 @@ async def test_cover(hass):
     assert "cover.deconz_old_brightness_cover" in gateway.deconz_ids
     assert "cover.window_covering_controller" in gateway.deconz_ids
     assert len(hass.states.async_all()) == 5
+    assert len(gateway.entities[cover.DOMAIN]) == 4
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6239266)